packages/cli/src/commands/promote/request-promote.ts

```diff
@@ -29,6 +31,13 @@ export default async function requestPromote({
     output: client.output,
   });
 
+  if (deployment.target !== 'production' && !force) {
+    output.error(
+      'Cannot promote deployment that does not target "production". If you are sure you want to do this, add `--force`.'
+    );
```

Comment:
It seems contradictory to say “cannot” and then say that you can with a flag. 

I think we can improve this message and include reasons why you wouldn’t want to do this. 

Something like:

> Warning: This deployment does not target production. Promoting will not apply production environment variables. Are you sure you want to continue? Y/n

Reply:
I like this better. In this framing, it does make more sense to use `--yes`.

Reply:
Switched! https://github.com/vercel/vercel/pull/10135/commits/de9914719cc5fca88e6977447550862eafa16c74